src/backend/src/services/work-packages.services.ts

```diff
@@ -34,7 +33,6 @@ export default class WorkPackagesService {
    * @returns a list of work packages
    */
   static async getAllWorkPackages(query: {
-    status?: WbsElementStatus;
```

Comment:
this stuff is definitely still needed, this is how we get work packages of a certain status